Controller/Adminhtml/Importcategory/Save.php

```diff
@@ -24,39 +24,61 @@
 * @license http://opensource.org/licenses/gpl-3.0.html GNU General Public License,version 3 (GPL-3.0)
 * @author Navin Bhudiya <navindbhudiya@gmail.com>
 */
+
 namespace Navin\ImportExportCategory\Controller\Adminhtml\Importcategory;
 
+use Magento\Catalog\Model\CategoryFactory;
 use Magento\Framework\App\Filesystem\DirectoryList;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Backend\Model\View\Result\Redirect;
+use Magento\Framework\File\Csv;
+use Magento\Framework\Filesystem;
+use Magento\Framework\Filesystem\Io\File;
+use Magento\Framework\Module\Dir\Reader;
+use Magento\Framework\Registry;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\DataObject;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Backend\Model\Session;
+use Magento\Backend\App\Action;
+use Magento\MediaStorage\Model\File\UploaderFactory;
+use Magento\Store\Model\StoreManagerInterface;
+use Psr\Log\LoggerInterface;
 
-class Save extends \Magento\Backend\App\Action
+class Save extends Action
 {
     /**
      * Backend session
      *
-     * @var \Magento\Backend\Model\Session
+     * @var Session
      */
     protected $_backendSession;
```

Comment:
Use consistent property naming conventions. Additionally [properties should not be prefixed with a single underscore to indicate private or protected visibility](https://www.php-fig.org/psr/psr-2/#42-properties).

Please ensure this is done for all variables.